### DISCUSSION TITLE
tabulator.exceptions.SourceError: field larger than field limit

### DISCUSSION THREAD

**Discussion by jeevangelista** [nih-cfde/published-documentation#335] [Bug] [answered]
I am getting this error when I validate my package. It seems to be related with the number of synonyms for a compound. I tried tweaking the preparation script and getting only a subset of synonyms fixed the issue. For reference, copper has more than 2000 synonyms: https://pubchem.ncbi.nlm.nih.gov/compound/23978#section=Depositor-Supplied-Synonyms&fullscreen=true

**Comment by abradyIGS** [comment]
I am looking into a systematic workaround for this issue: for now, stick with your stated workaround to get through validation. I may also wind up doing a (possibly brutal) truncation if the first few more palatable solutions don't work out. Leaving issue open; stand by

**Comment by abradyIGS** [comment, marked as answer]
I have truncated the synonym lists and hope this fixes your issue -- am marking "answered," but please reopen and report if you find otherwise.

Ontology files and the submission prep script have all been updated and published [here](https://osf.io/bq6k9/): please overwrite local copies and try again.

Note: the master C2M2 JSON schema has also updated (to match previous development schema): it is [here](https://osf.io/c63aw/) and an up-to-date copy needs to be included with submissions during prep with the script.
